app/operations/application_operation.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationOperation
+  def self.call(*args)
+    new(*args).call
```

Comment:
@sveredyuk 
Operations will be used only for controller actions (to clean controllers), when services will be created for another complex logic, api calls etc. It can keep simple logic and may be a something like wrapper for one or many services.
That will be more pleasantly to search actions.